src/Microsoft.CodeAnalysis.Analyzers/CSharp/Microsoft.CodeAnalysis.CSharp.Analyzers.csproj

```diff
@@ -9,4 +9,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.Analyzers.UnitTests" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
We used to automatically include C# and VB.NET workspace nugets, now I manually add the specific on the specific layer